crypto/asn1/a_time.c

```diff
@@ -489,7 +489,7 @@ int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm)
         if (tm->length > 15 && v[14] == '.') {
             f = &v[14];
             f_len = 1;
-            while (14 + f_len < l && isdigit(f[f_len]))
+            while (14 + f_len < l && ossl_isasciidigit(f[f_len]))
```

Comment:
I hadn't got to fixing them all yet.  I need to rethink the implementation to avoid the proliferation of character classes.

I think most (all) of the calls under asn1 will need to use the ASCII flavours.